app/models/invoice_subscription.rb

```diff
@@ -13,6 +13,15 @@ class InvoiceSubscription < ApplicationRecord
   monetize :subscription_amount_cents, disable_validation: true, allow_nil: true
   monetize :total_amount_cents, disable_validation: true, allow_nil: true
 
+  INVOICE_TYPE = {
```

Comment:
It will not be a real persisted type. Basically it means that the invoice itself can be related to two distinct subscription, one newly started and an other one terminated